### DISCUSSION TITLE
Documentation link broken

### DISCUSSION THREAD

**Discussion by nebbian** [pschatzmann/arduino-audio-tools#1554] [Q&A] [answered]
I'm trying to look for some documentation on this project, and came across this page:

https://github.com/pschatzmann/arduino-audio-tools/blob/main/docs/index.html

This mentions that the documentation is located here: https://pschatzmann.github.io/arduino-audio-tools/html/modules.html

Unfortunately this link is broken, it just shows a 404 "File not found" error page.

I just thought I'd report this.

**Comment by pschatzmann** [comment, marked as answer]
Why don't you just use the links provided in the [Readme](https://github.com/pschatzmann/arduino-audio-tools?tab=readme-ov-file#documentation)

**Comment by nebbian** [comment]
You're welcome :-)
